src/signals/incident/components/IncidentPreview/components/MapSelect/index.js

```diff
@@ -27,7 +26,7 @@ const MapSelectPreview = ({ value, endpoint, incident }) => (
   <Fragment>
     <Values>{value.join('; ')}</Values>
     <MapSelect
-      geojsonUrl={`${configuration.API_ROOT_MAPSERVER}${endpoint}`}
+      geojsonUrl={endpoint}
```

Comment:
Is this intentional, to remove the root?

Reply:
Yes, the whole endpoint URL comes from the config file